pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -64,14 +64,15 @@ core-packages:
 #
 # WARNING: This list is generated semiautomatically based on the most recent
 # LTS package set. If you want to add entries to it, you must do so before the
-# comment saying "# LTS Haskell x.y". Any changes after that commend will be
+# comment saying "# LTS Haskell x.y". Any changes after that comment will be
 # lost the next time `update-stackage.sh` runs.
 default-package-overrides:
   # This was only intended for ghc-7.0.4, and has very old deps, one hidden behind a flag
   - MissingH ==1.4.2.0
   # gi-gdkx11-4.x requires gtk-4.x, which is still under development and
   # not yet available in Nixpkgs
   - gi-gdkx11 < 4
+  - shelly ==1.8.1
```

Comment:
This shouldn't be here, but instead in the `extra-packages` section below.

We don't want to force the `haskellPackages.shelly` derivation to be version 1.8.1, but instead we want to generate a `haskellPackages.shelly_1_8_1` derivation in addition.

Reply:
Thanks, I have made this change and pushed.